core/src/test/scala/pureconfig/gen/package.scala

```diff
@@ -0,0 +1,66 @@
+package pureconfig
+
+import java.nio.file.{ Path, Paths }
+import java.time._
+import java.util.UUID
+
+import org.scalacheck.{ Arbitrary, Gen }
+import pureconfig.configurable.ConfigurableSuite
+import pureconfig.data._
+
+import scala.collection.JavaConverters._
+import scala.concurrent.duration.{ Duration, FiniteDuration }
+
+package object gen {
```

Comment:
47 degrees has a useful library of scalacheck generators which includes date/time generators. Should we consider using that instead of making our own ? 

Docs: https://47deg.github.io/scalacheck-toolbox/docs/datetime/index.html